homeassistant/components/modern_forms/strings.json

```diff
@@ -0,0 +1,28 @@
+{
+  "title": "Modern Forms",
```

Comment:
The title should not be translated, as it is a brand/product name in this case.
Please remove this line from the translations.